ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/stun.schema.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  stun:
+    type: dict
+    keys:
+      client:
+        type: dict
+        description: |
+          "If a device is a client"
+        keys:
+          server_profiles:
+            type: list
+            description: |
+              Enable Server profiles need to reference the name of the profile and IP address
+              Examples:
+              - "server-profile profile1
+                    ip addres 1.1.1.1"
+              - "server-profile profile2
+                    ip addres 2.2.2.2"
+            primary_key: name
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                ip_address:
+                  type: str
```

Comment:
```suggestion
                ip_address:
                  type: str
                  required: true
```

Reply:
```
leaf1(config-stun-server-profile)#show run | se stun
stun
   client
      refresh interval 42 seconds
      !
      server-profile BLAH
      !
```

ip_address is not required in EOS cli (4.30) so we should probably not enforce it from `eos_cli_config_gen` point of view (though I am not sure such server profile without ip address would be useful) - here is a glimpse of what could be implemented further

```
leaf1(config-stun-server-profile)#show cli commands
APPEND [ COMMENT ]
[no|default] comment
[no|default] disabled
do CMD
exit
[no|default] ip address IP_ADDR
[no|default] password profile PROFILE
[no|default] port PORT
show active
show active all [ detail ]
show comment
[no|default] ssl profile PROFILE
```